extensions/wikia/AdEngine/AdEngine.i18n.php

```diff
@@ -1,893 +1,585 @@
+	"adengine-exitstitial-button" => "דילוג על פרסומת זו",
+	"adengine-desc" => "מנוע הפרסום של ויקיה",
+];
+
+$messages["hrx"] = [
+	"adengine-exitstitial-title-template" => "Du verlässt $1 …",
+	"adengine-exitstitial-redirecting" => "Dein Browser leitet dich in weniche Sekunde uff die oongeforderte Seit weiter.",
+	"adengine-exitstitial-go-back" => "Retuar gehn?",
+	"adengine-exitstitial-button" => "Die Oonzeiche üwerspringe",
+];
+
+$messages["hu"] = [
+	"adengine-advertisement" => "hirdetés",
+	"adengine-exitstitial-redirecting" => "Böngésződ másodperceken belül átirányít a kívánt oldalra.",
+	"adengine-exitstitial-go-back" => "Visszamennél?",
+	"adengine-exitstitial-button" => "Hirdetés átugrása",
+];
+
+$messages["id"] = [
+	"adengine-advertisement" => "iklan",
+	"adengine-exitstitial-title-template" => "Anda meninggalkan $1...",
+	"adengine-exitstitial-redirecting" => "Peramban Anda akan mengarahkan Anda ke halaman yang Anda minta dalam beberapa detik.",
+	"adengine-exitstitial-go-back" => "Ingin kembali?",
+	"adengine-exitstitial-button" => "Lewati Iklan ini",
+	"adengine-desc" => "Mesin Iklan Wikia",
+];
+
+$messages["inh"] = [
+	"adengine-advertisement" => "дебат",
+];
+
+$messages["it"] = [
+	"adengine-advertisement" => "Pubblicità",
+	"adengine-exitstitial-title-template" => "Stai lasciando  $1 ...",
+	"adengine-exitstitial-redirecting" => "Il browser ti reindirizzerà alla pagina richiesta in pochi secondi.",
+	"adengine-exitstitial-go-back" => "Vuoi tornare indietro?",
+	"adengine-exitstitial-button" => "Ignora questo annuncio",
+	"adengine-desc" => "Wikia motore di ricerca",
+];
+
+$messages["ja"] = [
+	"adengine-advertisement" => "広告",
+	"adengine-exitstitial-button" => "広告をスキップ",
+	"adengine-exitstitial-title-template" => "You're leaving $1…",
+	"adengine-exitstitial-redirecting" => "Your browser will redirect you to the page you requested in a few seconds.",
+	"adengine-exitstitial-go-back" => "Want to go back?",
+	"adengine-desc" => "Wikia's Ad Engine",
+];
+
+$messages["ka"] = [
+	"adengine-exitstitial-title-template" => "თქვენ ტოვებენ $1...",
+	"adengine-exitstitial-redirecting" => "თქვენი ბრაუზერის გადამისამართება გაძლევთ თქვენს მიერ მოთხოვნილი გვერდი რამდენიმე წამში.",
+	"adengine-exitstitial-button" => "გამოტოვოთ ეს რეკლამა",
+];
+
+$messages["kn"] = [
+	"adengine-exitstitial-title-template" => "ನೀವು $1 ಬಿಡುತ ಇದ್ದೀರಿ.",
+	"adengine-exitstitial-redirecting" => "ನಿಮ್ಮ ಬ್ರೌಸರ್ ನೀವು ಕೇಳಿದ ಪುಟಕ್ಕೆ ನಿಮ್ಮನ ಸ್ವಲ್ಪ ಹೊತ್ತಲ್ಲೇ ಕಲಿಸುತದೆ.",
+	"adengine-exitstitial-go-back" => "ವಾಪುಸು ಹೋಗ ಬೇಕ?",
+	"adengine-exitstitial-button" => "ಈ ಎಡನ್ನು ಜಿಗಿ ಬೇಕ?",
+];
+
+$messages["ko"] = [
+	"adengine-advertisement" => "광고",
+	"adengine-exitstitial-title-template" => "$1 떠나는 중…",
+	"adengine-exitstitial-redirecting" => "브라우저가 몇 초 뒤에 요청한 페이지로 넘어갑니다.",
+	"adengine-exitstitial-go-back" => "돌아가시겠습니까?",
+	"adengine-exitstitial-button" => "광고 건너뛰기",
+	"adengine-desc" => "위키아 광고 엔진",
+];
+
+$messages["krc"] = [
+	"adengine-advertisement" => "реклама",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "реклама" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)